### ISSUE TITLE
"ereate dir" icon not tinted

### ISSUE THREAD

**Issue by simonpoole** [spacecowboy/NoNonsense-FilePicker#103]
Which is no surprise since currently it can't be done in the menu configuration. However it can be done programatically, best likely after inflating the menu. My current workaround https://github.com/MarcusWolschon/osmeditor4android/blob/master/src/de/blau/android/util/ThemedFilePickerActivity.java

**Comment by spacecowboy** [comment]
Yes well it's not supposed to be tinted :) Icons in the toolbar are typically white or dark grey, which can be seen in the Material Design guidelines: https://material.google.com

But in some designs it makes sense, so I'll add it as an example in the Wiki
